### PR TITLE
chore(README): Explain screenshots (#41)

### PR DESCRIPTION
**Pull request by sesam**
Issue #.
Adresses #41 

Changes:
Adds an initial explanation of the included screenshots.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,24 @@ Special thanks to:
...
 ![Login screen](assets/screenshot-1.png)
+
+The first screenshot shows the login screen. This plugin adds the SQRL login functionality on the login screen only.
+Registering happens automatically, but only when the WP site is allowing registrations. This is necessary to ensure the smooth first time visitor experience that SQRL users are expecting.
+
+---
+
 ![Profile screen without association](assets/screenshot-2.png)
+
+This second screenshot shows the user's profile when logged in with username and password, before having enable SQRL for log in.
```
Comment (alexhauser) [inline README.md]: I guess it should be

> ... before having enable**d** SQRL for **login**.

File: README.md [hunk 2]
```diff
@@ -18,6 +18,24 @@ Special thanks to:
...
+
+---
+
 ![Profile screen with association](assets/screenshot-3.png)
+
+This third screenshot shows the user's profile after having enable SQRL for log in.
```
Comment (alexhauser) [inline README.md]: Same here
> ... after having enable**d** SQRL for **login**.

File: README.md (hunk 1 above)
Comment (sesam) [inline README.md]:
```suggestion
This second screenshot shows the user's profile when logged in with username and password, before having enabled SQRL for log in.
```

File: README.md (hunk 2 above)
Comment (sesam) [inline README.md]:
```suggestion
This third screenshot shows the user's profile after having enabled SQRL for log in.
```

**Comment by sesam** [comment]
Good catch @alexhauser! I fixed it using the Github code change suggestion feature, now I see it made one commit each for those things, so I suggest a Squash-merge rather than normal merge once this is approved.

**Comment by alexhauser** [comment]
@sesam,

I'm not a native English speaker, so bear with me if I'm mistaken here, but to me this still seems wrong:
> ... before having enabled SQRL for **log in**.


To me, it seems it should either be

>... before having enabled SQRL for **login**.
or 
>... before having enabled SQRL for **logging in**.



After all, as a verb is's "to log in", but the noun for the procedure is "the login".

File: README.md [hunk 3]
```diff
@@ -18,6 +18,24 @@ Special thanks to:
...
 ![Login screen](assets/screenshot-1.png)
+
+The first screenshot shows the login screen. This plugin adds the SQRL login functionality on the login screen only.
+Registering happens automatically, but only when the WP site is allowing registrations. This is necessary to ensure the smooth first time visitor experience that SQRL users are expecting.
+
+---
+
 ![Profile screen without association](assets/screenshot-2.png)
+
+This second screenshot shows the user's profile when logged in with username and password, before having enabled SQRL for log in.
```
Comment (sesam) [inline README.md]:
```suggestion
This second screenshot shows the user's profile when logged in with username and password, as it looks before having enabled SQRL for logging in.
```

File: README.md [hunk 4]
```diff
@@ -18,6 +18,24 @@ Special thanks to:
...
+
 ![Profile screen without association](assets/screenshot-2.png)
+
+This second screenshot shows the user's profile when logged in with username and password, as it looks before having enabled SQRL for logging in.
+
+---
+
 ![Profile screen with association](assets/screenshot-3.png)
+
+This third screenshot shows the user's profile after having enabled SQRL for log in.
```
Comment (sesam) [inline README.md]:
```suggestion
This third screenshot shows the user's profile after having enabled SQRL for logging in.
```

**Comment by sesam** [comment]
Thank you @alexhauser - two occurences of this "Swenglish" fixed. Keep the fix requests coming :)

**Pull request merged**